SPECS/xcp-ng-xapi-storage.spec

```diff
@@ -1,23 +1,54 @@
 Name:           xcp-ng-xapi-storage
 Version:        1.1.0
-Release:        1%{?dist}
+Release:        1.0.zfsvol.0%{?dist}
 Summary:        XCP-ng implementation of the xapi-storage interface
 License:        LGPLv2.1
 URL:            https://github.com/xcp-ng/xcp-ng-xapi-storage
 Source0:        https://github.com/xcp-ng/xcp-ng-xapi-storage/archive/v%{version}/%{name}-%{version}.tar.gz
 
+Patch0:         xcp-ng-xapi-storage-1.1.0-zfs-zvol.patch
+
+BuildArch: noarch
+
 BuildRequires:  cmake3
 BuildRequires:  make
 BuildRequires:  python-setuptools
 
-Requires:       nbd
-Requires:       python-psutil
-Requires:       qemu-dp
-Requires:       systemd
+%description
+empty base package should not care about a description except it does
```

Comment:
This will be the description of the SRPM, AFAIK, so this looks like a good reason for caring and should be a real description.